src/TFTManager.cpp

```
@@ -0,0 +1,667 @@
+uint32_t TFTManager::hexToRgb(const char* hex) {
+    if (!hex || strlen(hex) < 6) return 0xCCCCCC;
+    char buf[7];
+    // Strip leading # if present
+    const char* h = (hex[0] == '#') ? hex + 1 : hex;
+    strncpy(buf, h, 6);
+    buf[6] = '\0';
+    unsigned long val = strtoul(buf, nullptr, 16);
+    return (uint32_t)val;
+}
+
+uint32_t TFTManager::dimColor(uint32_t color, uint8_t brightness) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)